### PR TITLE
Restore vertical nav bar layout, in particular label alignment

### PR DESCRIPTION
**Pull request by earshinov**
Fixes issues from https://github.com/madoar/angular-archwizard/pull/147#issuecomment-429548576

### CODE REVIEW

File: src/lib/components/wizard-navigation-bar.component.vertical.less [hunk 1]
```diff
@@ -243,7 +243,7 @@ aw-wizard-navigation-bar.vertical {
           .state-circle-with-border-hover(@dot-border-width, @wz-color-default);
         }
 
-        div {
+        a {
           min-height: @big-dot-height + 2 * @dot-border-width;
```
Comment (madoar) [inline src/lib/components/wizard-navigation-bar.component.vertical.less]: The height seems to be a little to large for the empty circle layout with and without symbol.
I believe that the `+ 2 * @dot-border-width` is not required anymore with the new div based step indicators.

Comment (madoar) [reply src/lib/components/wizard-navigation-bar.component.vertical.less]: I forgot to add an example image:
grafik
On the image you can see that the labels are slightly lower positioned than the symbols in the step indicators.

Comment (earshinov) [reply src/lib/components/wizard-navigation-bar.component.vertical.less]: Screenshot from the updated version (388a9777):

updated label alignment in vertical layout

**Comment by earshinov** [comment]
@madoar, I am not sure how well it will work out if we start using mixins and `:extend` together. But I definitely agree that there is too much duplication in style definitions. My counteroffer is to expand the use of mixins. I have spent some time and prepared an experimental version (for now only for the vertical layout) in https://github.com/madoar/angular-archwizard/pull/161. The `.less` file became considerably shorter (https://github.com/madoar/angular-archwizard/pull/161/commits/941775ab8e38d34e44ad6e46b9cacff40a58ee9a?utf8=%E2%9C%93&diff=split&w=1) while producing almost identical CSS. You can see this if you put the original `.less` file and my modified version into http://lesscss.org/less-preview/ and compare resulting CSS's with a diff tool of your choosing.

**Pull request merged**